test/unit/reductions/test_smoke.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import numpy as np
+import pandas as pd
+import platform
+import pytest
+from sklearn.linear_model import LogisticRegression
+from sklearn.svm import SVC
+from sklearn.tree import DecisionTreeClassifier
+
+from fairlearn.reductions import ExponentiatedGradient, GridSearch, DemographicParity, \
+    EqualizedOdds
+
+
+_ESTIMATORS = [LogisticRegression, SVC, DecisionTreeClassifier]
+
+if platform.system() != "Darwin":
+    # MacOS requires extra steps to install lightgbm properly, skipping for now
+    from lightgbm import LGBMClassifier
+    _ESTIMATORS.append(LGBMClassifier)
```

Comment:
You could also do `pytest.importorskip` in your test, to have it skipped if the env doesn't have the library. Looks nicer to me I guess.

Reply:
But then I can't parametrize the same way :-(

Reply:
If you find a way to solve this in a more elegant way using parametrization I can make a follow-up PR